src/components/analyses/listing/Listing.js

```diff
@@ -288,19 +305,19 @@ function Listing(props) {
     }, [currentNotification, updateAnalyses]);
 
     useEffect(() => {
-        setDetailsEnabled(selected && selected.length === 1);
+        setSingleSelection(selected && selected.length === 1);
     }, [selected]);
 
     useEffect(() => {
-        if (detailsOpen && data?.analyses) {
+        if (data?.analyses) {
             const selectedId = selected[0];
-            setDetailsAnalysis(
+            setSelectedAnalysis(
                 data.analyses.find((item) => item.id === selectedId)
             );
         } else {
-            setDetailsAnalysis(null);
+            setSelectedAnalysis(null);
         }
-    }, [data, detailsOpen, selected]);
+    }, [data, selected]);
```

Comment:
I started refactoring this `isSingleSelection` and `selectedAnalysis` state because I thought I would need them for enabling the rename menu item in the listing dot menu, but once I got to that part of the code I realized the other menu items are enabling and looking up analysis info in a different way.

I didn't bother undoing what I refactored so far, but I can still change it back.
I don't know, the code might be ever so slightly cleaner this way 🤷‍♂️

Reply:
`singleSelection` makes sense to me. We can refactor the rest of the code when encounter more changes / updates to `dot menu`.